docs/_templates/footer.html

```diff
@@ -0,0 +1,7 @@
+{% extends "!footer.html" %}
+{% block extrafooter %}
```

Comment:
Oh, I see, that's the block we're putting this in--not that it's trying to "block" having the extrafooter.